deploy/clowdapp.yaml

```diff
@@ -0,0 +1,174 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ccx-data-pipeline
+objects:
+
+- kind: HorizontalPodAutoscaler
+  apiVersion: autoscaling/v1
+  metadata:
+    labels:
+      app: ccx-data-pipeline
+    name: ccx-data-pipeline
```

Comment:
We already have HPA with the same name https://gitlab.cee.redhat.com/ccx/ccx-data-pipeline/-/blob/master/deploy/clowdapp.yaml#L13. Please use another one.